src/shared/index.ts

```diff
@@ -1 +1,4 @@
 export { useAuth, AuthUserContext, hasLoggedInToken } from "./lib";
+export { TOKEN_NAMES } from "./lib";
+export { getTokenFromCookie } from "./lib";
+export { saveTokenCookie } from "./ui";
```

Comment:
```suggestion
export { useAuth, 
  AuthUserContext, 
  hasLoggedInToken, 
  TOKEN_NAMES, 
  getTokenFromCookie,
  saveTokenCookie } from "./lib";
```

`export { saveTokenCookie } from "./ui";` - можешь файл `update-cookie.ts` переместить в папку `shared/lib/token`, чтобы все для работы с токенами было в одном месте. И реэкспорты тоже пробросить. Но надо будет проверить, что предыдущий код не поламался. Тот где используется saveTokenCookie.